cranelift/codegen/src/isa/zkasm/inst/mod.rs

```diff
@@ -1046,7 +1055,7 @@ impl Inst {
                 s
             }
 
-            &MInst::Extend {
```

Comment:
Not very understand why we switch from ``MInst`` to ``Inst``

Reply:
It was simply aliased renaming of the original `Inst` which didn’t seem particularly useful, and it being renamed didn’t help with `grep` either.